storybook/docker-compose.yml

```diff
@@ -1,20 +1,41 @@
 services:
+  composer-root:
+    image: composer
+    command: install
+    user: "${THE_UID}:${THE_GID}"
+    volumes:
+      - ./../:/autoupgrade
+    working_dir: /autoupgrade
+
+  composer-storybook:
+    image: composer
+    command: install
+    user: "${THE_UID}:${THE_GID}"
+    volumes:
+      - ./../:/autoupgrade
+    working_dir: /autoupgrade/storybook
+
   storybook-php:
-    build:
-      context: .
-      dockerfile: ./.docker/Dockerfile
-    command: symfony server:start
+    image: prestashop/base:8.2-apache
+    command: php -S 0.0.0.0:8000 -t public/
+    user: "${THE_UID}:${THE_GID}"
     volumes:
       - ./../:/autoupgrade
     working_dir: /autoupgrade/storybook
     ports:
       - "8003:8000"
+    depends_on:
+      composer-root:
+        condition: service_completed_successfully
+      composer-storybook:
+        condition: service_completed_successfully
 
   storybook-js:
     build:
       context: .
       dockerfile: ./.docker/Dockerfile
-    command: npm run storybook
+    command: bash -c "npm install && npm run storybook"
+    user: "${THE_UID}:${THE_GID}"
     environment:
       IS_IN_DOCKER: true
     volumes:
```

Comment:
Each time we start the container, the npm install and composer install commands on the 2 composer.json will be executed, I find it very tedious to just start a container, which should be quick after having installed once dependencies.

Reply:
It's not just about composer, the JS part is also reinstalled. If a template is modified/created, there is no need to reinstall the dependencies, because this is already configured in storybook, which directly points to the template directory. The impact is I think 10 to 20 seconds depending on the network. It's not much but if we can optimize it's better

Reply:
Afterwards it's just a small project for storybook, I'm not going to be stuck for that